packages/realm/src/Realm.ts

```diff
@@ -819,19 +826,19 @@ export class Realm {
   /**
    * Create a new {@link RealmObject} of the given type and with the specified properties. For objects marked asymmetric,
    * `undefined` is returned. The API for asymmetric objects is subject to changes in the future.
-   * @param type The type of Realm object to create.
-   * @param values Property values for all required properties without a
-   *   default value.
+   * @param type - The type of Realm object to create.
+   * @param values - Property values for all required properties without a
+   * default value.
    * @param mode Optional update mode. It can be one of the following values
-   *     - UpdateMode.Never: Objects are only created. If an existing object exists, an exception is thrown. This is the
-   *       default value.
-   *     - UpdateMode.All: If an existing object is found, all properties provided will be updated, any other properties will
-   *       remain unchanged.
-   *     - UpdateMode.Modified: If an existing object exists, only properties where the value has actually changed will be
-   *       updated. This improves notifications and server side performance but also have implications for how changes
-   *       across devices are merged. For most use cases, the behavior will match the intuitive behavior of how
-   *       changes should be merged, but if updating an entire object is considered an atomic operation, this mode
-   *       should not be used.
+   *   - UpdateMode.Never: Objects are only created. If an existing object exists, an exception is thrown. This is the
+   *     default value.
+   *   - UpdateMode.All: If an existing object is found, all properties provided will be updated, any other properties will
+   *     remain unchanged.
+   *   - UpdateMode.Modified: If an existing object exists, only properties where the value has actually changed will be
+   *     updated. This improves notifications and server side performance but also have implications for how changes
+   *     across devices are merged. For most use cases, the behavior will match the intuitive behavior of how
+   *     changes should be merged, but if updating an entire object is considered an atomic operation, this mode
+   *     should not be used.
    * @returns A {@link RealmObject} or `undefined` if the object is asymmetric.
    */
   create<T = DefaultObject>(
```

Comment:
The 3rd overload is missing the return type.

Reply:
Boah...this is more difficult to add than it seems 🤔

Reply:
Oh right, hmm.. Yeah that one's not clear cut how to do it. If you think including `undefined` for the last one works w/o breaking some TS code then let's do that.